contrib/go/tests/python/pants_test/contrib/go/targets/go_local_source_test_base.py

```diff
@@ -113,7 +114,8 @@ def test_globs_resources(self):
     # We should grab all of these though.
     self.create_file('src/go/src/foo/jake.go')
     self.create_file('src/go/src/foo/jake.png')
-    target = self.make_target(spec='src/go/src/foo', target_type=self.target_type)
+    self.add_to_build_file('src/go/src/foo', '{}()'.format(self.target_type.alias()))
+    target = self.target('src/go/src/foo')
```

Comment:
I think that some of the go code relies on synthetic targets working... but as long as the integration tests are still passing, this seems fine.